AlfrescoApp/View Controllers/Sync View Controller/Data Source/SearchCollectionViewDataSource.m

```diff
@@ -133,4 +133,9 @@ - (void)reloadDataSource
     [self retrieveNextItems:self.defaultListingContext];
 }
 
+- (NSString*)getSearchType
+{
+    return self.searchOptions.typeName;
```

Comment:
The case where searchOptions is nil should be handled here (perhaps by calling the parent method - see above comment).